### ISSUE TITLE
pointer_hide_delay isn't working anymore

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1769]
I just noticed that the `pointer_hide_delay` isn't working anymore. I have it on 2 seconds, but the pointer never hides. I confirmed it still worked in 19.1, I guess it's due to ImGui.

 In fullscreen, it always hides, even if I set it to -1. That was already the case in 19.1, but maybe it's better to not ignore this setting, for users who want to keep seeing the pointer?

**Comment by MBilderbeek** [comment]
We found out that ImGUI is also controlling the pointer, overriding the openMSX behaviour.

As experiment I added `	io.ConfigFlags |= ImGuiConfigFlags_NoMouseCursorChange;` to `ImGuiManager.cc` and that makes the openMSX behaviour work again.
Although it's a bit strange what happens to cursor visibility in full screen. When the cursor is /near/ the bottom or top, it seems to become visible already (I do have the status bar enabled). But well before it hits the status bar. Same happens at the top, but there the area is smaller where it already becomes visible.
